scss/helpers/_colored-links.scss

```diff
@@ -19,13 +19,13 @@
 
 // One-off special link helper as a bridge until v6
 .link-body-emphasis {
-  color: RGBA(var(--#{$prefix}emphasis-color-rgb), var(--#{$prefix}link-opacity, 1));
+  color: RGBA(var(--#{$prefix}emphasis-color-rgb), var(--#{$prefix}link-opacity));
   text-decoration-color: RGBA(var(--#{$prefix}emphasis-color-rgb), var(--#{$prefix}link-underline-opacity, 1));
 
   @if $link-shade-percentage != 0 {
     &:hover,
     &:focus {
-      color: RGBA(var(--#{$prefix}emphasis-color-rgb), var(--#{$prefix}link-opacity, .75));
+      color: RGBA(var(--#{$prefix}emphasis-color-rgb), var(--#{$prefix}link-opacity));
```

Comment:
I think this one needs to stay, or have it's `.75` value re-applied via CSS variable.

Reply:
Hey, sorry for the delay, I wasn't sure about this one since this is the only one having this behavior in https://main--twbs-bootstrap.netlify.app/docs/5.3/utilities/link/#colored-links. However, I might missunderstood something here and every link should shade ? In this case I need to change its CSS var with l.13 as well.

Reply:
We could perhaps add `--bs-link-opacity` variables here in addition to the color and text-decoration-color properties. Either works I think, provided the same functionality is available for overriding those with the new link utilities.

Reply:
wOh ok I think I can see my mistake here, I only compared the `.link-body-emphasis` to the `.link-dark` Should they behave the same ?

Reply:
Missed coming back to this one... `.link-body-emphasis` is a special case of the `.link-*` helpers. It's the only one that would get an alpha opacity override. Maybe drop the changes for the link opacity and just do the headings change?